tags/batchfile.md

```diff
@@ -0,0 +1,5 @@
+---
+name: batchfile
+displayName: Batch File
+description: Script file for running command line programs
+---
```

Comment:
Hi @Nitemice !

I noticed you are missing the text part of the tag file

See the other files in this folder for examples

Reply:
[`CONTRIBUTING.md`](https://github.com/Microsoft/opensource-tags/blob/master/CONTRIBUTING.md) seems to say that the "text part" is optional.
> Wiki content goes below this, and is optional for initial commits.

I'm happy to add something; probably just a copy of the description, as many others have done. I didn't originally because it doesn't really add any value and I didn't really know what else to say.

Reply:
oops. Thanks for the pointer and setting me straight :)